core-kotlin-modules/core-kotlin-collections-map/src/test/kotlin/com/baeldung/mergemap/MergeMapUnitTest.kt

```diff
@@ -0,0 +1,62 @@
+package com.baeldung.mergemap
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class MergeMapUnitTest {
+
+    @Test
+    fun givenTwoMaps_WhenMergeWithPlusOperator_ReturnsMergedMap() {
```

Comment:
Since we last talked I think the guidelines were updated for test naming in Kotlin, see here:
https://docs.google.com/document/d/1I6eu-eH5D1E0Mkl0Al-4xTgYiK2Ny0PPGtXaN2ToepI/edit

Now the preference is to have human readable test method names

Reply:
Thanks for sharing. I made these changes.